doc/ref/configuration/master.rst

```diff
@@ -1520,6 +1520,74 @@ Jinja environment init variable ``lstrip_blocks``.
 
     jinja_lstrip_blocks: False
 
+.. conf_master:: jinja_comment_prefix
+
+``jinja_line_comment_prefix``
+-----------------------
+
+.. versionadded:: 2017.FIXME
```

Comment:
applied.
btw any idea why the test are failing? They doesn't seem to fail because of this commit.